### PR TITLE
General section

### PR DESCRIPTION
**Pull request by jarronshih**

### CODE REVIEW

File: src/general/leading.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- 024th
+- Lead A or K
+- Honor leads: ace requests unblock or count signal; queen requests jack; highest equal from non-ace sequences and interior sequences
+
+
+## Later leading
+Against suit contracts:
+- Honor leads: king from ace-king; otherwise, highest equal from sequences and interior sequences
+- Spot-card leads in opening leader's suit: high from remaining doubleton; low from remaining tripleton
+- Spot-card leads in a new suit: third highest from even length; lowest from odd length
```
Comment (kunchiehw) [inline src/general/leading.md]: We lead attitude here or count here?

File: src/general/leading.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- Trump leads: count (low even, high odd, but preserve the highest spot card except to lead highest of equals)
+Against notrump contracts:
+- Honor leads: highest equal from sequences and interior sequences
+- Spot-card leads in opening leader's suit: original fourth highest
+- Spot-card leads in a new suit: attitude
+
+
+## Lead A or K
+### High level Contract
+- Lead A ask for encourage
```
Comment (kunchiehw) [inline src/general/leading.md]: encourage -> attitude

File: src/general/leading.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- Lead K ask for count
+
+When defensing a high level (over 5) contract,
+we need to cash AK if possible and distinguish the suit count.
+So when we lead with A, partner must tell us he has K or not.
+If lead with K, partner must tell us the count so we can decide to cash out or switch.
+
+
+## Partner's suit
+We lead `3rd/5th` in partner's suit so he could read our card count.
```
Comment (kunchiehw) [inline src/general/leading.md]: I have an agreement with lorderic before. Lets have a discussion over that later.

File: src/general/leading.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+
+## After Partner's Double
+### Overcall then Double
+Don't lead my suit.
+### Unusual Double on 3N
+Please lead declarer's suit or weird suit.
```
Comment (kunchiehw) [inline src/general/leading.md]: dummy's?

File: src/general/signal.md [hunk 5]
```diff
@@ -1,8 +1,35 @@
...
+## Signal Meanings
+### Partner's lead
+When following to partner's lead: attitude (at trick one, discouraging signal suggests the ability to support the obvious shift), but:
+- (?)suit-preference when a high honor is led and dummy has a singleton and can trump 
+- (?)count at trick one when not beating dummy's card or finessing and dummy's highest is the jack or lower or the equivalent
+
+### Declarer's or Dummy's lead
+When following to declarer's or dummy's lead: count (but see Smith, below)
+
+When following suit in trumps: suit-preference
```
Comment (kunchiehw) [inline src/general/signal.md]: Not sure we have agreement on this one.

**Review by kunchiehw — APPROVED** [review]

File: src/general/leading.md (hunk 1 above)
Comment (jarronshih) [reply src/general/leading.md]: attitude

File: src/general/leading.md (hunk 3 above)
Comment (jarronshih) [reply src/general/leading.md]: ok, mark as `?`

File: src/general/signal.md (hunk 5 above)
Comment (jarronshih) [reply src/general/signal.md]: should be count

**Pull request merged**